src/ariel-os-native/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "ariel-os-native"
+version = "0.1.0"
+license.workspace = true
+edition.workspace = true
+repository.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+cfg-if = { workspace = true }
+defmt = { workspace = true, optional = true }
+embassy-embedded-hal = { workspace = true, optional = true }
+embassy-executor = { workspace = true, default-features = false }
+embassy-time = { workspace = true, default-features = false, features = [
+  "std",
+] }
+embedded-hal-async = { workspace = true }
+paste = { workspace = true }
+portable-atomic = { workspace = true }
+ariel-os-debug = { workspace = true, features = ["std"] }
+ariel-os-embassy-common = { workspace = true }
+ariel-os-random = { workspace = true, optional = true }
+
+[features]
```

Comment:
These don't have any implementation behind, and are filtered by laze before they end up here. I kept them as scaffolding.